### ISSUE TITLE
ES2015 ol package: ol.interaction.Translate constructor fails

### ISSUE THREAD

**Issue by AnWeber** [openlayers/openlayers#6753]
With the new Release v4.0.1 the constructor call of ol.interaction.Translate throws an error. Version 4.0.1-beta2 is working.

###  Error
> angular.js:14525 TypeError: Cannot read property 'getChangeEventType' of undefined
    at new _ol_interaction_Translate_ (...)

### Source of Error
Source of Error was added with 5c23123. Code is working using [openlayers](https://www.npmjs.com/package/openlayers), but not with es2015 [ol](https://www.npmjs.com/package/ol/).

```
 _ol_events_.listen(this,
      _ol_.Object.getChangeEventType(_ol_.interaction.Property.ACTIVE),
      this.handleActiveChanged_, this);
 ```

#### Code

```
import olInteractionTranslate from 'ol/interaction/translate';
new olInteractionTranslate({
  features: ...
});
```

**Comment by AnWeber** [comment]
Thanks for the fast fix. I really appreciate the ES2015 packages. Great work.

**Comment by tschaub** [comment]
Thanks for the report @AnWeber.  The [`ol@4.1.1` release](https://www.npmjs.com/package/ol) includes the fix in #6755.  The same problem shouldn't crop up in future releases after #6770.  (And when we get remaining `closure-util` users to adapt, we can author in ES modules and avoid this problem altogether.)
